ingress-node-firewall/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the ingress-nodefw v1alpha1 API group
+//+kubebuilder:object:generate=true
+//+groupName=ingressnodefirewall.openshift.io
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "ingressnodefirewall.openshift.io", Version: "v1alpha1"}
```

Comment:
not aware of thing that can be grouped with this feature atm if u think its might be better to use general path like `network.openshift.io` pls let me know

Reply:
We currently lack a registry of chosen CR names, so having a significant group like network.openshift.io split into some things in openshift/api and some things in other repos, without any central registry is a problem for maintenance for the OCP networking team.  It's not pretty to have lots of small api groups, but it's better that conflicting schemas.

There is a PR from long ago where I floated an idea for a registry: https://github.com/openshift/api/pull/633

It could be built out, but it would take a fairly significant investment.

Reply:
@deads2k so you are saying stay they way we are for ingress node firewall APIs group right ?

Reply:
Yep, lets keep it as `ingressnodefirewall` for this API